### PR TITLE
Thommy

### PR DESCRIPTION
**Pull request by zelenikty**
Modified the redux code with layout, align-v and align-h functionality.
Code refactoring, mostly deletions.

### CODE REVIEW

File: org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts [hunk 1]
```diff
@@ -1,38 +1,27 @@
...
 import {AppActions} from '../redux/actions/app.actions';
-import {NgRedux, select} from '@angular-redux/store';
+import {NgRedux} from '@angular-redux/store';
 import {IAppState} from '../redux/store/app.store';
-import {Observable} from 'rxjs/Observable';
+import {ButtonsStateModel} from '../models/buttonsState.model';
+import {TopologyRendererActions} from '../redux/actions/topologyRenderer.actions';
 
 @Component({
   selector: 'app-canvas',
```
Comment (lharzenetter) [inline org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts]: The selector should be different than `app`.  I think we agreed on `winery` as well.
This should also go for **all** your components

File: org.eclipse.winery.topologymodeler.ui/src/app/app.component.ts [hunk 2]
```diff
@@ -206,6 +206,7 @@ export class AppComponent implements OnInit {
   constructor(private ngRedux: NgRedux<IAppState>, private actions: AppActions) {
   }
 
+
```
Comment (lharzenetter) [inline org.eclipse.winery.topologymodeler.ui/src/app/app.component.ts]: Can be deleted I think.

File: org.eclipse.winery.topologymodeler.ui/src/app/navbar/navbar.component.ts [hunk 3]
```diff
@@ -9,40 +9,36 @@
  * Contributors:
  *     Josip Ledic - initial API and implementation
```
Comment (lharzenetter) [inline org.eclipse.winery.topologymodeler.ui/src/app/navbar/navbar.component.ts]: If you've done some changes, you should insert your name and what you've done here.
This also goes for all files.

File: org.eclipse.winery.topologymodeler.ui/src/app/node/node.component.ts [hunk 4]
```diff
@@ -62,15 +50,11 @@ export class NodeComponent implements OnInit, AfterViewInit, DoCheck, OnDestroy
   @Output() addNodeToDragSelection: EventEmitter<any>;
   @Output() checkIfNodeInSelection: EventEmitter<string>;
   @Input() selectedNodes: any[] = [];
```
Comment (lharzenetter) [inline org.eclipse.winery.topologymodeler.ui/src/app/node/node.component.ts]: Please check if you really need `any` here. I think you know which type this variable should have.

Also: Please check this at **every** `any`. If no type can be applied, just give a short comment why you're using `any`.

File: org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts (hunk 1 above)
Comment (ledicjp) [inline org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts]: Currently TSLint complains about any other prefix than "app". You can try editing the _tslint.json_ by replacing the occurrences of "app" with "winery"

```"directive-selector": [true, "attribute", "app", "camelCase"],
"component-selector": [true, "element", "app", "kebab-case"],```

Comment (ledicjp) [inline org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts]: Currently TSLint complains about any other prefix than "app". You can try editing the tslint.json by replacing the occurrences of "app" with "winery"

"directive-selector": [true, "attribute", "app", "camelCase"],
"component-selector": [true, "element", "app", "kebab-case"],
I already set the prefix to "winery" in our angular-cli.json inside my pending pull request.

**Pull request closed without merging**